src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -2540,6 +2642,62 @@ struct MapperThruSections: MapperMaker
     }
 };
 
+TopoShape& TopoShape::makeElementSolid(const std::vector<TopoShape>& shapes, const char* op)
+{
+    return makeElementSolid(TopoShape().makeElementCompound(shapes), op);
```

Comment:
This one doesn't need the tag and hasher?

Reply:
Great question.  So, adding the standard ( 0, Hasher ) will not change the result - no elementMap,  Forcing a tag value  (99) will create an elementMap, but we can't do that here.

So, I suspect this whole call is bogus, and I avoided it in my tests and just stuck to what RT wrote.  I think it doesn't belong.  I'm open to any suggestions.

TIL Tag numbers are in hexadecimal.  99  became 63.

Reply:
As you'll see, I've transferred the code but commented it out.